### ISSUE TITLE
UVR GUI icon is barely visible on a dark taskbar

### ISSUE THREAD

**Issue by vsvyatski** [Anjok07/ultimatevocalremovergui#297]
I think you should redesign the icon that is shown on Windows taskbar. When I run UVR on Windows 10 with dark colored taskbar and the application is inactive, one can barely see its icon. See for yourself:

![icon on taskbar](https://user-images.githubusercontent.com/11542782/194565341-773a6232-18f2-45d1-a861-9b119e364972.png)

The problem is that the icon is in black color, it's a dark one, so it's evident that it's hardly distinguishable on the dark background. The problem is still reproducible in the latest 5.4.0 version. I hoped that you might have noticed the issue yourself (it's present for quite a while) and even tried to clear the Windows icon cache, but it didn't help.

**Comment by vsvyatski** [comment]
Well, it seems the 5.5.0 version fixes this. The GUI now uses the same icon as the launcher. But right after the installation of 5.5.0 the icon was kinda blurry. Then I went to the settings I noticed another patch (12_21_22), this one is not yet publicly exposed. After its installation the problem of blurriness was gone.

But I'll leave the liberty of closing the issue to the author, once he believes he's entirely done with this.
